chain-impl-mockchain/src/transaction/transfer.rs

```diff
@@ -189,6 +198,12 @@ pub struct Output<Address> {
     pub value: Value,
 }
 
+impl<Address: Readable> Output<Address> {
+    pub fn from(address: Address, value: Value) -> Self {
```

Comment:
This name is confusingly similar to `From::from`. Please name it either `new`, if it's the primary way to construct `Output`, or something descriptive like `from_address_and_value` if it's one of many constructors.